pkg/sql/sem/builtins/builtins_test.go

```diff
@@ -469,35 +469,6 @@ func TestExtractTimeSpanFromTimeTZ(t *testing.T) {
 	}
 }
 
-// TestResetIndexUsageStatsOnRemoteSQLNode asserts that the built-in for
-// resetting index usage statistics works when it's being set up on a remote
-// node via DistSQL.
-func TestResetIndexUsageStatsOnRemoteSQLNode(t *testing.T) {
```

Comment:
I'm unfamiliar with the terminology here - does remote node mean not gateway?

I don't think we have a test here to run `crdb_internal.reset_index_usage_stats()` on a gateway node either.